src/tests/suite/dfuse_test.c

```diff
@@ -48,6 +49,9 @@ print_usage()
 	print_message("dfuse_test -m|--metadata\n");
 	print_message("dfuse_test -d|--directory\n");
 	print_message("dfuse_test -l|--lowfd\n");
+	print_message("dfuse_test -e|--exec\n");
+	/* verifyenv is only run by exec test. Should not be executed directly */
+	/* print_message("dfuse_test -v|--verifyenv\n");                       */
```

Comment:
Is this is internal then only support the long option here.  -v could well use run by someone wanting to check the version.
```suggestion
	/* print_message("dfuse_test    --verifyenv\n");                       */
```

Reply:
Thank you! I will remove "-v".